vlan-httpks.ks.in

```diff
@@ -0,0 +1,86 @@
+#test name: vlan2-httpks
```

Comment:
The file is named `vlan-httpks` not `vlan2...`.

Reply:
Fixed